vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/Dashboard.java

```diff
@@ -272,23 +272,23 @@ public void setMinimumRowHeight(String minRowHeight) {
     }
 
     /**
-     * Returns the gap between the cells of the dashboard.
+     * Returns the spacing between the cells of the dashboard.
```

Comment:
It's no longer just the spacing between the cells but also the spacing around the layout (using padding internally)

Reply:
Updated the description to clarify what spacing does in the dashboard context.